### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by lordkroos12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	internal class Program
+	{
+		static void Main(string[] args)
+		{
+			Console.WriteLine("Enter a and b:");
+			int a = int.Parse(Console.ReadLine());
+			int b = int.Parse(Console.ReadLine());
+
+			Console.WriteLine($"In range [{a}, {b}] with exactly two 'A's in duodecimal representation:");
+
```
Comment (IgorKheidorov) [inline Program.cs:24]: a can be greater then b, they can be  negative. Please fix. case

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Cool in total, fix the comment

**Comment by lordkroos12** [comment]
Is it ok?

File: Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	internal class Program
+	{
+		static void Main(string[] args)
+		{
+			int a, b;
+			do
+			{
+				Console.WriteLine("Enter two positive integers a and b:");
+				a = int.Parse(Console.ReadLine());
+				b = int.Parse(Console.ReadLine());
```
Comment (IgorKheidorov) [inline Program.cs:12]: Negative values are allowed))), why not?

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**